docs/reference/schema-viewer-navigation.md

```diff
@@ -0,0 +1,169 @@
+| 🔵 Array[type]    	|  The `Array` element is a list containing other elements of a specific type e.g. strings, integers or objects. Square brackets `[]` are used to display array types in the schema browser. You'll find an example in the next section.              	|
+| ⚪️ Property(type) 	|   Properties, denoted with parentheses `()` are elements that contain data of a specific type e.g. string, boolean, interger, enumerations etc.        	|
+| ⚪️ <span style='color: orange'>Deprecated</span>     	|    Deprecated elements, while no longer recommended for use, still exist for backward compatibility with older systems or applications. Migration of deprecated elements is advised, as they may potentially be removed in newer versions, posing a risk to continued compatibility.        	|
+<!-- TODO add link to the example for object and array  --> 
+
+## Navigate the schema browser
+The Schema Browser offers two intuitive methods for navigation. First, you can easily search for a specific schema element using the search bar in the top bar. Simply enter the name or keyword of the element you're looking for, and the viewer will swiftly pinpoint and highlight the relevant part of the schema.
+
+![Image of schema viewer search](../images/schema-viewer-search.png)
+
+Once you've made a selection from the search results, the Schema Browser will automatically expand to reveal the corresponding schema object. Simultaneously, it will display informative contextual text associated with the selected element. This feature ensures that you not only find the element you were looking for but also gain a deeper understanding of its purpose and functionality, all within a single interaction.
+
+Alternatively, you can navigate the Schema Browser by interacting directly with the visual representation. Each schema element is denoted by a bullet. By clicking on the blue bullets, you can easily expand the tree structure to reveal the hierarchical relationships. Clicking (1) on the bullet text (e.g. JdbcTableDataObject) triggers the immediate display of the corresponding schema information (2).
+
+![Image of schema viewer information](../images/schema-viewer-information.png)
+
+## Mapping schema to config file
+
+Generally speaking you can take this as a reference, the naming convention for elements in the configuration file generally mirrors the nomenclature of the elements in the schema browser, with some notable omissions. These omissions include the element specifications such as `{}`, `[mapOf]`, `[oneOf]`, `[]`, or `(type)`. Let's delve into the specifics of each element designation.
+
+When an element's name concludes with `{}`, it signifies an object capable of possessing properties and nested elements that require a designated key.
+
+Example from schema browser (schema{}->global{}):
```

Comment:
Why so much code for such short dependency. I suggest to reduce the examples to the facts listed for the schema browser. As done in the execution mode example below.

Reply:
Sorry I don't understand this point. Could you eventually add a short example?

Reply:
Fixed in commit 0f4aba20db7f870f10381c58427679336fe9353c. Thanks for the oral clarification today.